src/Roave/SecurityAdvisories/FriendsOfPhp.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Roave.SecurityAdvisories.FriendsOfPhp (
+  parseAdvisoryYaml,
+  Advisory(..),
+  AffectedBranch(..)
+) where
+
+import Data.Map (Map)
+import Data.ByteString (ByteString)
+import Data.Text (Text)
+import Data.Yaml
+import qualified Data.Yaml as Y
+
+data AffectedBranch = AffectedBranch {
+  time :: Maybe Text
+  , versions :: [Text]
```

Comment:
`[Text]` is obviously too simplistic, but it will do until we decide to go further into detail

Reply:
From what I read, I will need to implement my own `FromJSON` instance once I defined the specification for a `VersionBoundary`

Reply:
Yes. Good for now (except for the formatting of course :P)